common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -792,6 +792,9 @@
     public static final KnownKey zimbra_class_jsieve_comparators_octet = KnownKey.newKey("com.zimbra.cs.filter.ZimbraOctet");
     public static final KnownKey zimbra_class_two_factor_auth_factory = KnownKey.newKey("com.zimbra.cs.account.auth.twofactor.TwoFactorAuth$DefaultFactory");
 
+    // ZCS-8181 if below flag is false, do not update zimbraAppSpecificPassword attr
+    //          with last authentication time during authentication of app specific password.
+    public static final KnownKey zimbra_two_factor_apppassword_authtime = KnownKey.newKey(true);
```

Comment:
Please change the name to include update auth time

Reply:
Done